platform/wismote/dev/acc-sensor.c

```diff
@@ -42,8 +42,8 @@ static void
 activate(void)
 {
   /* This assumes that some other sensor system already did setup the ADC
-  /* (in the case of the sky platform it is sensors_light_init that does it)
-
+   * (in the case of the sky platform it is sensors_light_init that does it) */
+  #if 0
```

Comment:
Oh, I'm sorry. I just scanned over the changes. Of course the `#if 0` is much better than the block-comment!